app/components/questionnaire/fastq-file-pair-row.js

```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
+import FileGroupRow from 'bespin-ui/components/questionnaire/file-group-row';
+
+const FASTQFilePairRow = FileGroupRow.extend({
+  classNames: ['fastq-file-pair-row', 'well','well-sm'],
+  pairName: Ember.computed.oneWay('pair.name'),
+  errors: Ember.computed('pair.file1', 'pair.file2', 'pair.name', function() {
+    const errors = [];
+    const file1 = this.get('pair.file1');
+    const file2 = this.get('pair.file2');
+    const name = this.get('pair.name');
+    let message = '';
```

Comment:
You could add a test for the following code that generates different error messages.